plover/steno_dictionary.py

```diff
@@ -245,6 +251,15 @@ def casereverse_lookup(self, value):
             if key:
                 return key
 
+    def regex_reverse_lookup(self, pattern, match_limit=10):
+        try:
+            r = re.compile(pattern)
+            matches = set(itertools.chain.from_iterable((d.regex_value_match(r) for d in self.dicts if d.enabled)))
+            mkpairs = ((m,self.reverse_lookup(m)) for m in sorted(matches))
+            return [(m,k) for (m,k) in mkpairs if k][:match_limit]
+        except re.error as e:
+            return [("Invalid regular expression", [(e.msg,)])]
```

Comment:
That error handling does not look right: exception handling should be done in the caller.